### PR TITLE
Update feature request template

### PR DESCRIPTION
**Pull request by shaobo-he-aws**
*Issue #, if available:*

*Description of changes:* Make feature request template cedar-spec specific.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: Feature request
-description: Suggest an idea for Cedar
+description: Suggest an idea for Cedar specifications
```
Comment (khieta) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]:
```suggestion
description: Suggest an idea for the Cedar specification
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -11,14 +11,13 @@ body:
   - type: dropdown
     attributes:
       label: Category
-      description: What component of Cedar does this feature relate to?
+      description: What component of Cedar specifications does this feature relate to?
```
Comment (khieta) [inline .github/ISSUE_TEMPLATE/feature_request.yml:14]:
```suggestion
      description: What component of the Cedar specification does this feature relate to?
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -11,14 +11,13 @@ body:
...
       multiple: true
       options:
-        - Cedar language features
-        - Cedar validation features
-        - Cedar grammar changes/additions
-        - User level API changes
-        - New DRT target
+        - Dafny definitional engine
```
Comment (khieta) [inline .github/ISSUE_TEMPLATE/feature_request.yml:17]:
```suggestion
        - Dafny formalization
```

**Review by cdisselkoen — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -11,14 +11,13 @@ body:
...
+        - DRT targets
```
Comment (khieta) [inline .github/ISSUE_TEMPLATE/feature_request.yml:18]:
```suggestion
        - DRT target(s)
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 5]
```diff
@@ -11,14 +11,13 @@ body:
...
+        - DRT generators
```
Comment (khieta) [inline .github/ISSUE_TEMPLATE/feature_request.yml:19]:
```suggestion
        - DRT generator(s)
```

**Review by khieta — APPROVED** [review]
Lgtm, but I left several (minor) comments because I was excited to try out the "suggestion" review feature.

**Review by andrewmwells-amazon — APPROVED** [review]

**Comment by shaobo-he-aws** [comment]
> Lgtm, but I left several (minor) comments because I was excited to try out the "suggestion" review feature.

Thank you. Now I could try the "commit suggestion" feature :)

**Review by andrewmwells-amazon — APPROVED** [review]

**Pull request merged**